admin/workspace-management/tun-device.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TUN device
+description: Learn how to enable TUN devices for VPN usage within Coder.
+state: alpha
+---
+
+By default Coder workspaces don't contain a TUN device for security reasons,
```

Comment:
```suggestion
By default, Coder workspaces don't contain a TUN device for security reasons,
```

Is this actually true, security reasons? I don't think it should be a security issue, and writing this without context about the possible security risks may scare people away from using it

Reply:
True, I guess I should remove the security bit